internal/controller/volumereplicationgroup_controller.go

```diff
@@ -2275,8 +2261,6 @@ func (r *VolumeReplicationGroupReconciler) RGSMapFunc(ctx context.Context, obj c
 
 	rgs, ok := obj.(*ramendrv1alpha1.ReplicationGroupSource)
 	if !ok {
-		log.Info("map function received not a replication group source resource")
```

Comment:
We should drop other such log lines in the functions above this. While I have never seen this log, and so will not contribute to reducing log noise as such, it is not essential as well.